SH4RE/SH4RE/ViewListing/ViewListingView.swift

```diff
@@ -147,12 +148,27 @@ struct ViewListingView: View {
         .frame(alignment: .bottom)
         .onAppear(){
             self.price = listing.price
+            self.numberOfImages = listing.imagepath.count
+            for path in listing.imagepath{
+                let storageRef = Storage.storage().reference(withPath: path)
+                //Download in Memory with a Maximum Size of 1MB (1 * 1024 * 1024 Bytes):
+                storageRef.getData(maxSize: 1 * 1024 * 1024) { [self] data, error in
```

Comment:
preview for `ViewListingView` breaks on this line, is it because the imagepath doesn't match anything for the preview?